src/app.tsx

```diff
@@ -1,8 +1,12 @@
 import { AppShell, useMantineColorScheme } from "@mantine/core";
+import { Amplify } from 'aws-amplify';
 import { Outlet } from "react-router-dom";
+import config from './amplifyconfiguration.json'; // amplify v6 Ref: https://docs.amplify.aws/javascript/build-a-backend/auth/set-up-auth/
```

Comment:
<img width="644" alt="image" src="https://github.com/surcyf123/pretraining-frontend/assets/25290212/27aee0fe-8c15-45c3-bbf6-30d77ce81c5f">

As  per v6 of amplify we should use json

Reply:
https://docs.amplify.aws/javascript/build-a-backend/auth/set-up-auth/